app/components/app_simple_status_banner_component.rb

```diff
@@ -50,8 +55,16 @@ def full_name
     @patient_session.patient.full_name
   end
 
-  def triage_nurse
-    most_recent_triage&.user&.full_name
+  def nurse
+    most_recent_event =
+      if most_recent_triage&.created_at.to_i >
+           most_recent_vaccination&.created_at.to_i
+        most_recent_triage
+      else
+        most_recent_vaccination
+      end
```

Comment:
Golf is popular up in Scotland, isn't it?

```suggestion
      [most_recent_triage, most_recent_vaccination].compact.max_by(
        &:created_at
      )
```

Assuming you can compare the `created_at` directly without the `to_i`, which I would've thought you could?

Reply:
Thanks, I wanted to golf something like this but my brain wasn't working at the time!